stratum/hal/lib/barefoot/bf_chassis_manager.cc

```diff
@@ -264,7 +264,7 @@ ::util::Status BfChassisManager::PushChassisConfig(
   // new maps
   std::map<int, uint64> unit_to_node_id;
   std::map<uint64, int> node_id_to_unit;
-  std::map<uint64, std::map<uint32, PortState>>
+  std::map<uint64, std::map<uint32, OperStatus>>
```

Comment:
I feel that adding a new map for the `last_change` value might be less disruptive.